Clinical Key.js

```diff
@@ -0,0 +1,190 @@
+{
+	"translatorID": "a55463ba-e403-415b-80d4-284d5f9b4b15",
+	"label": "Clinical Key",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https://(www\\.|www-)clinicalkey(\\.|-)com",
```

Comment:
Typical URLs are either www.clinicalkey.com (or its proxied version) or the hyphenated proxy, www-clinicalkey-com.proxy.example.com. As @dstillman noted above, hyphenated proxies aren't adjusted, so this is just a workaround until that is resolved